components/rhs_thread/rhs_thread.jsx

```diff
@@ -453,6 +454,7 @@ export default class RhsThread extends React.Component {
                             isBusy={this.state.isBusy}
                             isEmbedVisible={this.props.postsEmbedVisibleObj[selected.id]}
                         />
+                        {isFakeDeletedPost && <DateSeparator date={rootPostDay}/>}
```

Comment:
When I have a FakeDeletedPost I still use the `rootPostDay` as date separator? I don't know if we need to use the something like `firstNotDeletedPostDay`.

Reply:
@jespino I'll check what the mobile app is using. If they're not the same I'll make a change to one of them so that they're consistent.

Reply:
@jespino Just verified that it's using the date of the replies in both cases, so no change required.